pkg/identityserver/application_access.go

```diff
@@ -67,7 +70,22 @@ func (is *IdentityServer) createApplicationAPIKey(ctx context.Context, req *ttnp
 	}
 	key.Key = token
 	events.Publish(evtCreateApplicationAPIKey(ctx, req.ApplicationIdentifiers, nil))
-	// TODO: Send notification email (https://github.com/TheThingsNetwork/lorawan-stack/issues/72).
+	members := map[*ttnpb.OrganizationOrUserIdentifiers]*ttnpb.Rights{}
+	err = is.withDatabase(ctx, func(db *gorm.DB) (err error) {
+		members, err = store.GetMembershipStore(db).FindMembers(ctx, req.EntityIdentifiers())
+		return err
+	})
+	for member := range members {
+		if userIDs := member.GetUserIDs(); userIDs != nil {
+			err = is.SendUserEmail(ctx, userIDs, func(data emails.Data) email.MessageData {
+				data.SetEntity(req.EntityIdentifiers())
+				return &emails.APIKeyCreated{Data: data, Key: key}
```

Comment:
I think we should not send the entire `key` object to the template. That would allow network operators to define templates that send the actual plaintext key in an email, which shouldn't be possible.